src/default.ts

```diff
@@ -0,0 +1,2 @@
+export const DEFAULT_DSCVR_API_URL =
+  'https://edge1-analytics.stg.dscvr.cloud/graphql';
```

Comment:
should this be publicly exposed?

Reply:
good question - @jimmymadrigal this should be `https://dev.dscvr.one/graphql`. It's not setup yet, but for commit purposes let's please use that.

Reply:
Fixed